backend/quadcore/restful/linkedin/auth.py

```diff
@@ -10,11 +10,13 @@ def get(self):
         auth_info = LinkedIn.get_auth_info(access_token)
 
         if auth_info != None:
-            session["username"] = auth_info["username"]
+            session["temp_name"] = auth_info["temp_name"]
             session["email"] = auth_info["email"]
             session["linkedin_token"] = access_token
 
-            if DataManager.is_exist_user(auth_info["email"]):
+            username = DataManager.check_email_username(auth_info["email"])
+            if not username == None:
```

Comment:
This code makes me confused. :) How about `username != None` or `username is not None`? I recommend the first one because of the overall convention of this project.

Reply:
oh .. Ok! I modified.